docs/source/conf.py

```diff
@@ -28,7 +28,7 @@
 # The short X.Y version
 version = ''
 # The full version, including alpha/beta/rc tags
-release = '0.0.2'
+release = '1.0.0'
 
```

Comment:
@juliakreutzer do you mean `release = 1.0.0`, not `version = 1.0.0`, right?

Reply:
I think both of them could be `1.0`, what do you think? :-)

Reply:
Yes! Sorry @may- I meant the release of course. And we can ditch the last zero @bastings :smile:

Reply:
Yeah, not just for my sake, but it looks like `version` requires format X.Y so 1.0 would work for both `version` and `release`.
Release can be `1.0.0` if you like :D

Reply:
@may- could you please update this accordingly?